schemas/hash.yml

```diff
@@ -39,3 +43,13 @@
       level: extended
       type: keyword
       description: SHA512 hash.
+
+    - name: ssdeep
+      level: extended
+      type: keyword
+      short: SSDEEP hash.
+      description: >
+        SSDEEP hash.
+
+        Learn more at https://ssdeep-project.github.io/ssdeep/index.html.
```

Comment:
This link is broken due to the included `.`.

However, I suggest removing the link entirely. It's the top search hit for `ssdeep`, and removing it will make for one less link to maintain in the ECS docs.

WDYT?